google/cloud/spanner/samples/samples.cc

```diff
@@ -3802,6 +3802,16 @@ void QueryInformationSchemaDatabaseOptions(
 }
 // [END spanner_query_information_schema_database_options]
 
+// [START spanner_set_batching_delay]
```

Comment:
This tag does not appear in the published list.  If we're making up a new one, it should at least include "commit".

Reply:
I've changed the tag. This is new - is there somewhere I need to add it that I've missed?

Reply:
Firstly, it looks like you only updated the START tag, and not the END one.

But to the bigger issue, the point of these tags is to delimit regions that will be scraped into the Cloud Spanner documentation, so introducing a tag requires some cross-team coordination.  For example, here are all the defined Spanner tags containing "commit": http://devrel/snippets?pageSize=100&sort=region_tag&direction=asc&product_id=9&region_tag=%25commit%25

Normally the tech writers would introduce the tag when they document the feature, otherwise it has no purpose.